### DISCUSSION TITLE
Encoding and resulting file name

### DISCUSSION THREAD

**Discussion by AedBlanquer** [enzo1982/freac#527] [Help & Support]
Hello everyone.

I'm using FR:AC to convert from mp3 to ogg as a xhole (in french : "en masse").

In "General settings/Output files", I have defined "<track>" as the resulting file name.
However, when encoding, FR:AC systematically suggests saving the resulting file as "<ALBUMARTIST> - <ALBUM>".
What haven't I understood?
What do I need to correct?

Best regard
André Blanquer

**Comment by AedBlanquer** [comment]
Sorry: as a whole (in french: "en masse")

**Reply by Lee-Carre** [reply]
At least in British English (or perhaps just Jersey (in the Channel Islands)), we use “en masse” fairly often.

To my understanding, the closest English equivalent would be ‘in bulk’, though, that doesn't have quite the same meaning, and I prefer the French term.

---

Separately; you're able to edit your posts on GitHub (it saves the revision history); in the top right of your original post, the ‘three horizontal dots’ (ellipsis: …), clicking that should show a menu, with one of the entries being ‘edit’. This then lets you fix typos, other errors, or add more information.

**Comment by AedBlanquer** [comment]
I finally get it!
The names of tags in different ripping or tag management software are not necessarily the same... even in English.
Sorry for the unnecessary reading.

Best regards.
André Blanquer

**Reply by Lakrits** [reply]
@AedBlanquer : For future reference, you can consult the fre:ac manual's section on filename patterns, either in the programme's in-built help, or online at https://www.freac.org/manual/en/howto.html#anchor_pat.

The start page for the online manual is https://www.freac.org/manual/index_en.html.

Et, si vous souhaitez nous aider, n'hésitez pas à traduire le manuel en français. :)

**Comment by Lee-Carre** [comment]
> convert from mp3 to ogg as a [w]hole (in french : "en masse").

Ogg is a container format (for bit-streams of codec data, among other things). I presume that you mean either Vorbis or Opus as the actual codec.

> In "General settings/Output files", I have defined "" as the resulting file name.

An empty/null string for the filename pattern would mean ‘no filename’, which doesn't make sense, and I'm not sure is actually possible.

If you're wanting the filename unchanged (except the extension (from `.mp3` to `.ogg`/`.oga`/`.opus`), then there is a configuration checkbox to enable that. I forget what it's called, from memory; something about using (for output) the same filename as the input file.

The pattern syntax (in my experience, and to my knowlede) is more-so when extracting from CDs. The files have to be named something, and the audio on the CDDA doesn't have filenames. Hence generating the filenames of the encoded audio from the metadata.

> However, when encoding, FR:AC systematically suggests saving the resulting file as " - ".

Because hyphens are the default separator between different metadata. As mentioned, the filename can't be empty/null, so it must use something; since you've told it to not use the input filename, or artist-name & track-name, then I can well imagine how this would result.

> What haven't I understood?
> What do I need to correct?

Like Lakrits said, use the filename pattern syntax to tell Fre:AC how to name output files. Lots of options, customise it however you wish. Look in the drop-down menu for where the pattern is input, to see other examples (outside of the documentation). Else, when the input is another file (MP3 in your case), you probably want the option to use the same name as a base (but it'll have to change the extension).

**Reply by AedBlanquer** [reply]
Think you for your clear answer.

Cordially
André Blanquer
